src/components/modal-backdrop.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+
+const ModalBackdrop = (props) => {
+  const {
+    className,
+    handleExitModal,
+    ...otherProps
+  } = props;
+
+  const bodayTag = document.getElementsByTagName('body')[0];
```

Comment:
It can't go in the `useEffect()` function, otherwise it will add the class name to the body element on each document update. Used `document.body` and fixed the typo as well!

Reply:
Yeah, my comment isn't that helpful as the only way the component can update is if it closes. For future reference though, you can use `useEffect` to run code only once by supplying an empty array as a second argument. I made a quick sandbox to demo this: https://codesandbox.io/s/dank-night-1ebhx?fontsize=14&hidenavigation=1&theme=dark

Reply:
Great, thanks for the heads-up! I didn't like this so much when I was learning about hooks that I had totally forgot about it!